docs/ext/rfc.py

```diff
@@ -24,7 +24,7 @@
 import re
 
 
-RFC_PATTERN = re.compile('\RFC (\d{4}), Section ([\d\.]+)')
```

Comment:
Slightly off-topic, but found this typo while working on these docs, and went ahead and fixed it.